app/views/favorites/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Your Favorite Pets</h1>
+<section>
+  <% if favorite.total_count > 0 %>
```

Comment:
I can't think of where else this logic should live, but maybe something to follow up with during Monday's checkin

Reply:
ditto... I would like to revisit this during check-in to find out if there are better syntax options or other approaches for sure...